collector/src/main/java/com/bitmovin/analytics/utils/HttpClient.java

```diff
@@ -26,7 +26,11 @@ public HttpClient(Context context, String url) {
         this.context = context;
     }
 
+
     public void post(String postBody, final Callback callback) {
+        if (client == null) {
```

Comment:
Why do we do this here? Is there any benefit to late-initializing the `OkHTTPClient`

Reply:
Because of how Player unit Tests work. :)